libs/blocks/table-of-contents/table-of-contents.css

```diff
@@ -103,8 +125,8 @@
 
   .section.toc .marquee .foreground.container {
     margin-right: 0;
-    flex-wrap: wrap;
-    align-content: center;
+    align-content: space-between;
```

Comment:
@JasonHowellSlavin  Should this be set to `align-content: start` to completely close the gap between the text and the image?

I'm currently still seeing a small gap due to `space-between`

![image](https://user-images.githubusercontent.com/2539954/226451220-5e747034-c337-4164-892c-639e4b09e9f0.png)

Reply:
Good looking out. There will be a small gap because the image itself should be flush with the bottom of the container and the ToC content is much larger than the image and text combined height-wise. By replacing the hard coded `gap` with `space-between` we reduce the gap fairly significantly. Striving for parity with https://business.adobe.com/solutions/customer-journeys.html.